src/daemon/manager.py

```
@@ -151,3 +160,182 @@ def get_all_jobs_status() -> Dict[str, Any]:
+                        return {
+                            "success": True,
+                            "jobs": jobs_data,
+                            "total_jobs": len(job_states),
+                            "timestamp": datetime.now().isoformat(),
+                            "source": "mongodb_fallback"
+                        }
+                    
+                    mongo_client.close()
+                else:
+                    mongo_client.close()
+            except Exception as mongo_e:
```

Ruff (0.12.2)

Local variable `mongo_e` is assigned to but never used

Remove assignment to unused variable `mongo_e`

(F841)